process-engine/bpmn-studio
# ✨ Create Own Pagination

## Changes

1. Create Own Pagination

## Issues

Closes #1791 

PR: #1805

## How to test the changes

- Check if the pagination is still working for the `process-list` (dashboard), `task-list` (dashboard), `cronjob-list` (dashboard) and `correlation-list` (inspect-correlation).
